backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java

```diff
@@ -1043,7 +1043,12 @@ private void writeCpuPinningPolicyMetadata() {
         if (vm.getVmPinning() != null && cpuPinningPolicy == CpuPinningPolicy.NONE) {
             cpuPinningPolicy = CpuPinningPolicy.MANUAL;
         }
-        writer.writeElement(OVIRT_VM_URI, "cpuPolicy", cpuPinningPolicy.name().toLowerCase());
+        String cpuPinningPolicyName = cpuPinningPolicy.name().toLowerCase();
+        if (cpuPinningPolicy == CpuPinningPolicy.ISOLATE_THREADS) {
+            // in VDSM the policy is `isolate-threads` while in JAVA ENUM can't be set with a dash.
```

Comment:
It's a pity we didn't catch this nuisance during API discussions.

Reply:
Yes, this is unfortunate.